datamechanics_airflow_plugin/operator.py

```diff
@@ -1,5 +1,8 @@
+import json
 import time
 
+from typing import Dict, Optional, Union
```

Comment:
Thanks for introducing typing in the package :)

Reply:
You're welcome!